### PR TITLE
Feature/kitosudv 3322

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Tests.Unit.Core.ApplicationServices/ApplicationServices/Handlers/AuthorizedUpdateOrganizationFromFKOrganisationCommandHandlerTest.cs [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+
+        [Fact]
+        public void Execute_Performs_Update_Of_Existing_Synchronized_Tree()
+        {
+            //Arrange
+            var organization = new Organization();
+            organization.OrgUnits.Add(CreateOrganizationUnit(organization, true)); //Add the root
+            const int oldDepth = 2;
+            const int newDepth = 3;
+            organization.StsOrganizationConnection = new StsOrganizationConnection
```
Comment (mrjsawdk) [inline Tests.Unit.Core.ApplicationServices/ApplicationServices/Handlers/AuthorizedUpdateOrganizationFromFKOrganisationCommandHandlerTest.cs:48]: all of these tests were migrated from sts service and adjusted

File: Tests.Unit.Core.ApplicationServices/ApplicationServices/Organizations/StsOrganizationSynchronizationServiceTest.cs [hunk 2]
```diff
@@ -303,114 +304,6 @@ public void Connect_Hierarchy_Fails_If_GetOrganization_Fails()
...
-            SetupResolveOrganizationTreeReturns(organization, externalRoot);
-            var transaction = ExpectTransaction();
-
-            //Act
-            var error = _sut.UpdateConnection(organizationId, newDepth, false);
-
-            //Assert
-            Assert.False(error.HasValue);
-            Assert.NotNull(organization.StsOrganizationConnection);
-            Assert.True(organization.StsOrganizationConnection.Connected);
```
Comment (mrjsawdk) [inline Tests.Unit.Core.ApplicationServices/ApplicationServices/Organizations/StsOrganizationSynchronizationServiceTest.cs:348]: moved to command handler test

**Review by a-naskret — APPROVED** [review]

File: Core.DomainModel/Organization/Organization.cs [hunk 3]
```diff
@@ -486,5 +486,20 @@ private static bool MatchRoot(OrganizationUnit unit)
         {
             return unit.Parent == null;
         }
+
+        public Maybe<OperationError> UnsubscribeFromAutomaticUpdates(OrganizationUnitOrigin origin)
+        {
```
Comment (mrjsawdk) [inline Core.DomainModel/Organization/Organization.cs:491]: this should use the getconnection

**Comment by mrjsawdk** [comment]
integration build https://teamcity.strongminds.dk/viewLog.html?buildId=11056&tab=buildResultsDiv&buildTypeId=Os2Kitos_DeployToIntegration

**Pull request merged**